src/main/java/io/numaproj/numaflow/sink/Response.java

```diff
@@ -1,12 +1,21 @@
 package io.numaproj.numaflow.sink;
 
+import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @Getter
-@AllArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class Response {
     private final String id;
     private final Boolean success;
     private final String err;
+
+    public static Response responseOK(String id) {
+        return new Response(id, true, null);
+    }
+
+    public static Response responseFailure(String id, String errMsg) {
```

Comment:
Is `responseFailure` used anywhere? If not, shall we look into covering it in unit tests?